sdk/src/client/node_api/mqtt/error.rs

```diff
@@ -18,3 +20,24 @@ pub enum Error {
     #[error("invalid topic {0}")]
     InvalidTopic(String),
 }
+
+// Serialize type with Display error.
+impl Serialize for Error {
```

Comment:
probably would be easier to define a struct here with serialize derived

Reply:
I could actually remove it again. Added it for a different impl of error handling we had earlier.